neurodamus/metype.py

```diff
@@ -208,6 +208,32 @@ def __init__(self, gid, cell):
         self.gid = gid
 
 
+class PointCell:
+    """Class representing a minimal, single section cell for dry-runs"""
+
+    def __init__(self, gid, cell_info, _circuit_conf):
+        self.gid = gid
+        self.raw_gid = None
+        self.soma = [Nd.Section(name="soma[0]", cell=self)]
+        self.exc_mini_frequency = float(cell_info.exc_mini_frequency)
+        self.inh_mini_frequency = float(cell_info.inh_mini_frequency)
+        self.synHelperList = []
+        self.synlist = []
+
+    CellRef = property(lambda self: self)
+    CCell = property(lambda self: self)
+    nSecAll = property(lambda _self: 1)
+    all = property(lambda self: self.soma)
+    input_resistance = property(lambda _self: 1)
```

Comment:
Is `input_resistance` set for stimulus? If so, do we also need to set `threshold_current` and `holding_current` ?

Reply:
You have a point to have this API more complete. I clearly did the minimal changes only.
`threshold_current` and `holding_current` don't seem to be cells properties but rather of `METypeItem` metadata objects. However cell objects seem to commonly rely on `getThreshold` function. Let me add the common functions from `METype` then